### PR TITLE
Improve particles

### PR DESCRIPTION
**Pull request by Lizard-13**
I've fixed the non-random PIXI particles speed, I just noticed the minimumSpeedMultiplier :/
So the real differences right now:
* PIXI particles looking direction is affected by gravity, Cocos and native particles direction is affected by speed/force only.
* Some Cocos particles systems need the maximum particles on screen > 150
* ~Cocos particles size seems bigger than PIXI and native.~ EDIT: Fixed!
* Cocos allocates memory for the particles when you create the system, if the maximum particles on screen is pretty big (>5000) it can pause the game a split of a second each time you create a particle system instance.

### CODE REVIEW

File: Extensions/ParticleSystem/particleemitterobject-pixi-renderer.js [hunk 1]
```diff
@@ -235,9 +218,10 @@ gdjs.ParticleEmitterObjectPixiRenderer.prototype.setAlpha = function(alpha1, alp
     }
 };
 
-gdjs.ParticleEmitterObjectPixiRenderer.prototype.setFlow = function(flow){
+gdjs.ParticleEmitterObjectPixiRenderer.prototype.setFlow = function(flow, tank){
     this.emitter.frequency = flow < 0 ? 0.0001 : 1.0/flow;
-    // TODO: This should also affect emitter lifetime.
+    this.emitterLifetime = tank < 0 ? -1 :
```
Comment (4ian) [inline Extensions/ParticleSystem/particleemitterobject-pixi-renderer.js]: Thanks for fixing this! :)

**Comment by 4ian** [comment]
> Cocos allocates memory for the particles when you create the system, if the maximum particles on screen is pretty big (>5000) it can pause the game a split of a second each time you create a particle system instance.

I think we should add a warning somewhere (on the wiki, or in the IDE editor) that using more than a few hundreds particles (let's say 200) is a bad idea anyway :)

**Comment by 4ian** [comment]
Looks good to me! Thanks for taking the time to fix all these things :)

**Comment by 4ian** [comment]
A note here to notify you that I've released GD5 beta 31. It would be great to make GD5 support the particles extension. It's something that I plan to do later, unless someone when to try this before (basically, it consists in compiling GDevelop.js, with the Particles extension included, and then make the editor for particles objects)

**Comment by Lizard-13** [comment]
I was checking the new IDE code today, I have to figure out how everything fits together, but I'll try to do it :)

**Pull request merged**